aoe4/formatters.js

```diff
@@ -124,13 +129,20 @@ class NightbotDefaultFormatter {
 
     const rank = mode.rank ? `rank #${mode.rank}` : 'unranked';
     const rank_level = mode.rank_level ? formatRankLevel(mode.rank_level) + ', ' : '';
-    var msg = `${player.name} is ${rank} (${rank_level}${mode.rating} Elo), with ${mode.games_count} game${mode.games_count == 1?'':'s'} (${mode.wins_count}-${mode.losses_count} | ${mode.win_rate}%)`;
+    const rating = player.rating ? ` (${rank_level}${mode.rating} Elo)` : '';
+    const games = mode.games_count ? mode.games_count : "no";
+    var msg = `${player.name} is ${rank}${rating}, with ${games} game${mode.games_count == 1?'':'s'} (${mode.wins_count?mode.wins_count:0}-${mode.losses_count?mode.losses_count:0} | ${mode.win_rate?mode.win_rate:0}%)`;
 
     if (Number.isInteger(mode.streak)) {
         const streak = parseInt(mode.streak) < 0 ? "losing" : "win";
         msg += `, on a ${Math.abs(mode.streak)}-game ${streak} streak`;
     }
-    msg += ` [last played ${formatAge(mode.last_game_at)} ago]`;
+
+    if (mode.last_game) {
+        msg += ` [last played ${formatAge(mode.last_game_at)} ago]`;
+    } else {
+        msg += " [never played]";
```

Comment:
do we need this, or can we just leave it out entirely? It'd kinda redundant.

Reply:
Removed the `else`